src/server/index.js

```diff
@@ -1,5 +1,3 @@
-const serverConfig = require('./config');
```

Comment:
I think it was more readable this way than to directly call something on the `require`'s result.